.prow.yaml

```
@@ -0,0 +1,13 @@
+presubmits:
+  - name: pre-commit
+    decorate: true
+    skip_report: false
+    always_run: true
+    context: pre-commit
+    spec:
+      containers:
+        - image: quay.io/thoth-station/thoth-precommit-py38:v0.13.0
```

The latest version I see [in quay](https://quay.io/repository/thoth-station/thoth-precommit-py38?tab=tags&tag=latest) is:

```suggestion
        - image: quay.io/thoth-station/thoth-precommit-py38:v0.14.3
```